graphql/schema/schema.graphql

```diff
@@ -148,6 +155,8 @@ type Query {
   allStudios: [Studio!]!
   allMovies: [Movie!]!
   allTags: [Tag!]!
+  allGalleries: [Gallery!]!
+  allImages: [Image!]!
```

Comment:
I don't think these interfaces are a good idea, and I'm not sure why they are added.

In any case, you need to add implementations for these resolves to `queryResolvers`. See `AllTags` in `resolver_query_find_tag.go`. The typical workflow is to just implement them with a `panic("not implemented")` until you're ready to implement them properly.

This explains this error message:

```
D:\Projets\stashStar\internal\api\resolver.go:57:9: cannot use &(queryResolver literal) (value of type *queryResolver) as QueryResolver value in return statement: *queryResolver does not implement QueryResolver (missing method AllGalleries)
```
There should be one for `AllImages` as well. Not sure why there isn't.

Reply:
> I would suggest first splitting this PR up. You're adding new fields to the image object, which needs its own PR.

This is because I want to make a scrap by URL directly. If I do a second pr for this, I will then need a third one to meet the two end.
Same goes for the date. I want scrappers to be able to grab said date

Reply:
Right, but adding these two fields is a significant PR alone. It's easier for me to review and test two separate PRs, one for adding the fields, and another for scraping, than it is to review one giant PR that does everything.

The way I would do it would be to create a branch/PR for adding the fields first. When that is complete, submit that PR, and if you want to start work on scraping, create a new branch off the PR branch to do so. You can rebase off `develop` when the first PR is merged.